compass/compass.js

```diff
@@ -0,0 +1,53 @@
+﻿API.onUpdate.connect(function (sender) {
+
+    if (0.3 < API.getGameplayCamDir().X && 0.3 < API.getGameplayCamDir().Y) {
+
+        API.drawText("NE", 350, 1000, 1, 255, 255, 255, 255, 2, 1, false, true, 0);

```

Comment:
Of course, but you'd only need to have the `API.drawText` line once (at the end of the function).